src/templates/base/2019/base.html

```diff
@@ -99,11 +99,134 @@
 <li><a href="{{ url_for('contributors', year=year, lang=lang) }}">{{ self.contributors_title() }}</a></li>
 <li><a href="{{ url_for('methodology', year=year, lang=lang) }}">{{ self.methodology_title() }}</a></li>
 {% endblock %}
 
+{% macro nav_table_of_contents(switcher_name) %}
+  {% if year == '2019' %}
+    {% if switcher_name == "mobile" or switcher_name == "mobile-footer" %}
+      {{ native_table_of_contents_dropdown(switcher_name) }}
+    {% else %}
+      {{ styled_table_of_contents_dropdown(switcher_name) }}
+    {% endif %}
+  {% endif %}
+{% endmacro %}
+
+{% macro native_table_of_contents_dropdown(switcher_name) %}
+{% set current_title = metadata.get('title') if metadata else None %}
+<div class="table-of-contents-switcher">
+  <label for="table-of-contents-switcher-{{switcher_name}}" class="visually-hidden">
+    {{ self.table_of_contents_title() }}
+  </label>
+  <select id="language-switcher-{{switcher_name}}">
+    <option disabled selected value> {{ self.table_of_contents_title() }} </option>
+    <option value="{{ url_for('table_of_contents', year=year, lang=lang) }}">{{ self.foreword_title() }}</option>
+    {% for part_config in config.outline %}
+    {% for chapter_config in part_config.chapters %}
+      {% set title = localizedChapterTitles[chapter_config.slug] if localizedChapterTitles[chapter_config.slug]|length else chapter_config.title %}
+      {% if chapter_config.todo %}
+        <option disabled>
+          {{ self.chapter() }} {{ chapter_config.chapter }}: {{title}}
+        </option>
+      {% else %}
+        {% if chapter_lang_exists(lang, year, chapter_config.slug) %}
+          {% if current_title == title %}
+            <option selected value="{{ url_for('chapter', year=year, lang=lang, chapter=chapter_config.slug) }}">
+              {{ self.chapter() }} {{ chapter_config.chapter }}: {{title}}
+            </option>
+          {% else %}
+            <option value="{{ url_for('chapter', year=year, lang=lang, chapter=chapter_config.slug) }}">
+              {{ self.chapter() }} {{ chapter_config.chapter }}: {{title}}
+            </option>
+          {% endif %}
+        {% else %}
+        {% if current_title == title %}
+          <option selected value="{{ url_for('chapter', year=year, lang='en', chapter=chapter_config.slug) }}">
+            {{ self.chapter() }} {{ chapter_config.chapter }}: {{title}}
+          </option>
+        {% else %}
+          <option value="{{ url_for('chapter', year=year, lang='en', chapter=chapter_config.slug) }}">
+            {{ self.chapter() }} {{ chapter_config.chapter }}: {{title}}
+          </option>
+        {% endif %}
+        {% endif %}
+      {% endif %}
+    {% endfor %}
+    {% endfor %}
+
+    <option value="{{ url_for('methodology', year=year, lang=lang) }}">
+      {{ self.methodology_title() }}
+    </option>
+    <option value="{{ url_for('contributors', year=year, lang=lang) }}">
+      {{ self.contributors_title() }}
+    </option>
```

Comment:
I found a why to handle the currently selected item. It's a little extra code but think worth is as you do it for the chapters:

```suggestion
    {% if request.path.endswith("methodology") %}
    <option selected disabled value="{{ url_for('methodology', year=year, lang=lang) }}">
      {{ self.methodology_title() }}
    </option>
    {% else %}
    <option value="{{ url_for('methodology', year=year, lang=lang) }}">
      {{ self.methodology_title() }}
    </option>
    {% endif %}
    {% if request.path.endswith("contributors") %}
    <option selected disabled value="{{ url_for('contributors', year=year, lang=lang) }}">
      {{ self.contributors_title() }}
    </option>
    {% else %}
    <option value="{{ url_for('contributors', year=year, lang=lang) }}">
      {{ self.contributors_title() }}
    </option>
    {% endif %}
```

Reply:
this is the solution I've been looking for. Thank you!